sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -158,6 +167,57 @@ public FormRecognizerServiceVersion getServiceVersion() {
             fetchExtractReceiptResult(includeTextDetails));
     }
 
+
+    /**
+     * List all available models.
+     *
+     * @return {@link PagedFlux} of {@link CustomFormModelInfo}.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedFlux<CustomFormModelInfo> listModels() {
+         try {
+            return new PagedFlux<>(() -> withContext(context -> listFirstPageModelInfo(context)),
+                continuationToken -> withContext(context -> listNextPageModelInfo(continuationToken, context)));
+        } catch (RuntimeException ex) {
+            return new PagedFlux<>(() -> monoError(logger, ex));
+        }
+    }
+
+    PagedFlux<CustomFormModelInfo> listModels(Context context) {
```

Comment:
ServiceMethod annotation.

Reply:
it is a helper method that used in Sync Client.

Reply:
`@ServiceMethod(returns = ReturnType.COLLECTION)`

Reply:
This is package-private helper method, it should not need to include @ServiceMethod, will follow up this with u offline.